bundles/org.openhab.binding.telegram/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -114,4 +118,64 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="messageEvent">
```

Comment:
I would suggest to mark all 4 new channel types as `advanced`, since trigger channels aren't normally meant to carry data structures in JSON. For the use case at hand I understand that this is helpful, though, so I'm fine with it.

Reply:
DONE I made them all advanced.